test/library/input.spec.js

```diff
@@ -56,6 +56,33 @@ describe('NumberFormat as input', () => {
     expect(wrapper.find('input').instance().value).toEqual('$2,456,981');
   });
 
+  it('should load the default value when initial value is null', () => {
+    const wrapper = mount(<NumberFormat value={null} defaultValue={89} />);
+    expect(wrapper.state().value).toEqual('89');
```

Comment:
We should never check for state value. As it's implementation details. Instead we should test for the end side-effect.
```js
expect(wrapper.find('input').instance().value).toEqual('89');
```